source/class.cpp

```diff
@@ -910,6 +911,11 @@ string bind_constructor(ConstructorBindingInfo const &CBI, uint args_to_bind, bo
 
 		//string params = args_to_bind ? ", " + args.first : "";
 		string params = args_to_bind ? args.first : "";
+		
+		string argsHelper;
+		for(uint i=0; i<CBI.T->getNumParams()  and  i < args_to_bind; ++i) {
+			argsHelper += ", pybind11::arg(\"{}\")"_format( string( CBI.T->getParamDecl(i)->getName() ) );
+		}
```

Comment:
@lyskov  I reverted the change in `function.cpp`. my main purpose of this PR was to get documentation of the arguments for other constructors in Python environment that is mainly here. And does not need that change  in `function.cpp`.